project_stages.md

```diff
@@ -0,0 +1,176 @@
+## Becoming an OpenSSF Project
+
+The OpenSSF's mission is to inspire and enable the community to secure the open source software we all depend on. OpenSSF is focused on the security of existing open source project however, there are missing pieces in the tooling needed to improve project security. These pieces are the projects OpenSSF aims to steward and support. 
+
+# Project Life Cycle
+
+New projects to the OpenSSF, and progression through the project lifecycle, are approved by the Technical Advisory Committee (TAC). Projects oversight is provided by either the TAC (if project is independent from a WG) or a WG (if project is part of a specific WG and follows that WG's governance). 
+
+Projects follow the Sandbox, Incubating, Graduated and Archival lifecycle stages defined below. Projects that seek widespread adoption and end user use are expected to progress through the stages. 
+
+<!-- TOC -->
+* [Sandbox](#sandbox)
+* [Incubating](#incubating)
+* [Graduated](#graduated)
+* [Archived](#archived)
+<!-- /TOC -->
+
+
+A project's lifecycle stage is a signal of maturity level. Projects progress through the lifecycle by demonstrating their maturity in technical vision, impact to the open source community, and sustainability in governance and contributors. All projects, at all stages, are expected to abide by the OpenSSF's Code of Conduct. 
+
+
+### Sandbox
+
+The OpenSSF Sandbox is the entry point for early stage projects and has four goals:
+
+* Encourage experimentation and open collaboration that can add value to the OpenSSF mission and build the ingredients of a successful Incubation level Project
+* Facilitate alignment with existing Working Groups or Projects if this is desired
+* Nurture Projects
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to OpenSSF legal, code of conduct, and IP Policy requirements.
+
+#### Project Responsibilities
+
+* Bi-annual updates to the TAC on technical vision and progress on vision
+* Maintain a diversified contributor base (i.e. not a single-vendor project)
+* Provides project updates to OpenSSF Marketing WG as requested
```

Comment:
```suggestion
* Provides project updates to OpenSSF Marketing Committee as requested
```

Reply:
@AevaOnline What is the difference between the OpenSSF Committee and the OpenSSF WG, is it defined anywhere ?